src/main/java/org/kainos/ea/db/ConnectionDAO.java

```diff
@@ -0,0 +1,34 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.client.DatabaseConnectionFailedException;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+public class ConnectionDAO {
+
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+
+
+    public String testDatabaseConnection() throws DatabaseConnectionFailedException {
+
+        try (Connection c = databaseConnector.getConnection()) {
```

Comment:
`getConnection` is static, but you are calling it in a way that means `getConnection` doesn't need to be static

Reply:
Done